public/js/definitions-html5-css.js

```diff
@@ -0,0 +1,91 @@
+$(document).ready(function(){
```

Comment:
I think I'm starting to get it, would I need to start the file with something like

```
window.common = (function(global) {
  const {
    $,
    Rx: { Observable },
    common = { init: [] },
    Mousetrap
  } = global;
```
?